src/app/components/component-viewer/component-viewer.component.ts

```diff
@@ -22,12 +22,15 @@ export class ComponentViewerComponent implements OnInit {
             .subscribe(docItem => {
                 this.docItem = docItem;
 
-                let availableSections = ['API'];
+                let availableSections: string[] = [];
 
                 if (this.docItem) {
                     const examples = this.docItem.examples;
                     if (examples && examples.length > 0) {
-                        availableSections = ['Examples', 'API'];
+                        availableSections = ['Examples'];
+                    }
+                    if (!this.docItem.hideApi) {
+                        availableSections.push('API');
```

Comment:
Because of the folder structure for the pipes, the dgeni process doesn't generate API documentation for pipes.  We may want to change this in the future, but the API surface for pipes is very small and pretty consistent (just a transform function that accepts a value).  I instead document the API in the usage markdown file.